vistrails/packages/uvcdat_cdms/init.py

```diff
@@ -1953,8 +1953,8 @@ def get_input_ports(plot_type):
     elif plot_type == "Isoline":
         return expand_port_specs([('label', 'basic:String', True),
                                   ('levels', 'basic:List', True),
-                                  ('ext_1', 'basic:String', True),
-                                  ('ext_2', 'basic:String', True),
+                                  ('ext_1', 'basic:Boolean', True),
+                                  ('ext_2', 'basic:Boolean', True),
```

Comment:
hum... not that it matters really but isolines do NOT have ext_1 and ext_2... Not sure why this bit is here